_posts/2017-03-27-open source update.md

```diff
@@ -0,0 +1,95 @@
+
+Open source software development has been critical at CFPB since we opened our doors. 
+In April 2012, just nine months after the Bureau opened, 
+we released our [Open Source Policy](http://www.consumerfinance.gov/blog/the-cfpbs-source-code-policy-open-and-shared/), 
+based on the work of the [Department of Defense](http://dodcio.defense.gov/OpenSourceSoftwareFAQ.aspx#OSS_and_DoD_Policy), 
+along with our first two open source projects. 
+Just six days later, 
+[we accepted our first pull request from a member of the public](https://github.com/cfpb/transit_subsidy/pull/1).
+
+Our approach to open source in these early days 
+was to publicly release small bits of source code that had potential for wider application. 
+In November of 2012 and January of 2013, 
+our first class of design and development fellows began working at the Bureau. 
+Two of the projects that were developed by this initial class of fellows, 
+the open data platform [Qu](https://github.com/cfpb/qu) 
+and [eRegulations](https://github.com/cfpb/eRegulations), set a new precedent: 
+they were developed from day one entirely in the open on GitHub. 
+Shortly afterwards, we formed an open source working group, 
+which worked to put procedures in place for open sourcing code that was only available internally. 
+As more code was moved to the public, the team moved towards a policy of open source by default, 
+where source code is developed in private only in certain 
+[well-defined cases](https://github.com/cfpb/source-code-policy/blob/gh-pages/cfpb-source-code-policy.txt#L81-L94), 
+including security risks or when the license is restricted.
+
+With an open source by default policy in place, 
+we’ve been able to collaborate with other government agencies 
+and the public at hackathon events where designers and developers come together 
+to work on open source software:
+
+- CFPB staff have been present at National Day of Civic Hacking events 
+in [2013](http://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking/), 
+[2014](http://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking-again/),  
+and [2015](http://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking-2015/), 
+working with developers to create data visualizations using the [Consumer Complaint Database](http://www.consumerfinance.gov/complaintdatabase/) 
+and [Home Mortgage Disclosure Act](http://www.consumerfinance.gov/hmda) data.
+- Software developers from 18F’s eRegs team attended the General Services Administration’s 
+[GovTechHack](https://18f.gsa.gov/2015/04/02/govtechhack-hacking-for-civic-improvement/) in 2015, 
+where they focused on improving the software’s local setup process. 
+They succeeded in decreasing the setup time for eRegs from 2 hours to 15 minutes.
+- CFPB Developer Catherine Farman gathered a team of designers and coders 
```

Comment:
you're asking here for the word "coders" on line 68 to change to "developers," right?